src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -49,12 +49,11 @@ export default function GamesSettings({ useDetails = true }: Props) {
   const { platform } = useContext(ContextProvider)
   const { isDefault, gameInfo, isLinuxNative } = useContext(SettingsContext)
   const [wineVersion] = useSetting('wineVersion', defaultWineVersion)
+  const [nativeGame, setNativeGame] = React.useState(false)
```

Comment:
is this change needed? we use `useContext` and `useEffect` directly, for consistency I would argue to use `useState` directly without `React.`

Reply:
The change is needed, will fix the import